static/js/index.js

```diff
@@ -18,10 +97,61 @@ burger.addEventListener('click', () => {
     iwq.classList.add('show');
   }
 });
-// ====================================================================================
+// ============================================================================
+// eslint-disable-next-line no-unused-vars
+function detail(c) {
+  parpopup.style.display = 'flex';
+  const update = details.filter((item) => item.category === c);
+  parpopup__popup.innerHTML = `
+  <div class="head">
+  <span class="name">${update[0].title}</span>
+  <span id="close" onclick="closes()">&times;</span>
+</div>
+<ul>
+  <li><a href="">${update[0].list.list1}</a></li>
+  <li><a href="">${update[0].list.list2}</a></li>
+  <li><a href="">${update[0].list.list3}</a></li>
+</ul>
+<div class="popup__responsive">
+  <img src="${update[0].image}" alt="" class="responsive" />
+</div>
```

Comment:
- [ ] Kindly add some padding/ margin top to give some spacing between your image and the text sections above it. 

**This is how I am seeing your design right now:**

![image](https://user-images.githubusercontent.com/86318284/195418360-730e8eba-e69e-4dbf-8465-0c2cc8906682.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/195418413-327cf36f-c285-468b-8627-93937297ebc5.png)